frontend/patches/@stitches+react+1.2.6.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/node_modules/@stitches/react/dist/index.cjs b/node_modules/@stitches/react/dist/index.cjs
```

Comment:
Why do we need to patch it? 🤔

Reply:
Oh yeah, forgot to mention this one: https://github.com/modulz/stitches/issues/908
I'll open a PR in their repo later.